### PR TITLE
NF 573 - asynch file transfer

### PR DESCRIPTION
**Pull request by rorticus0502**

### CODE REVIEW

File: app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/FileTransferAudit.scala [hunk 1]
```diff
@@ -16,23 +16,14 @@
...
-                              transferSuccess: Option[Boolean] = None,
-                              transferHttpStatus: Option[Int] = None,
-                              transferredAt: Option[LocalDateTime] = None,
-                              transferError: Option[String] = None
+case class FileTransferAudit(caseReferenceNumber: String, fileTransferResults: Seq[FileTransferResult]
                             )
 
 object FileTransferAudit {
   implicit val formats: Format[FileTransferAudit] = Json.format[FileTransferAudit]
+//  implicit val resultWrites: Writes[FileTransferAudit] = Json.writes[FileTransferAudit]
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/FileTransferAudit.scala]: Can delete line 28

File: app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/responses/NDRCCaseResponse.scala [hunk 2]
```diff
@@ -20,10 +20,11 @@ import play.api.libs.json.{Format, Json}
 
 case class NDRCCaseResponse(
                                        correlationId: String,
+                                       caseId: Option[String],
                                        error: Option[ApiError] = None,
                                        result: Option[NDRCFileTransferResult] = None
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/responses/NDRCCaseResponse.scala]: Just checking - is`result: Option[NDRCFileTransferResult]` still needed?

**Review by SteveSugden — COMMENTED** [review]
LGTM - just a question and minor comment

File: app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/responses/NDRCCaseResponse.scala (hunk 2 above)
Comment (rorticus0502) [reply app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/responses/NDRCCaseResponse.scala]: It is not - just deleted it and I am running the tests now.

**Review by SteveSugden — APPROVED** [review]
LGTM

**Pull request merged**